kodilla-good-patterns/src/test/java/com/kodilla/good/patterns/airport/FlightSearchEngineTestSuite.java

```diff
@@ -1,38 +1,57 @@
 package com.kodilla.good.patterns.airport;
 
+import org.junit.Assert;
 import org.junit.Test;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class FlightSearchEngineTestSuite {
     @Test
     public void testFindFlightFrom() {
         //Given
-        FlightInitializer flights = new FlightInitializer();
-        FlightSearchEngine flightSearchEngine = new FlightSearchEngine(flights.initFlights());
+        FlightInitializer initializedFlights = new FlightInitializer();
```

Comment:
a nie lepiej by to bylo nazwac poprostu flightInitializer? Jesli zmienna ma w nazwie flights, to oczekuje jakichs tam lotow, a tu przeciez lotow nie ma ;)